.github/workflows/gitleaks-gitleaks-action-v2.yml

```
@@ -19,7 +19,7 @@ permissions: read-all
 jobs:
   v2:
     name: v2
-    uses: ericcornelissen/reproducing-actions/.github/workflows/reusable-test.yml@main
+    uses: ericcornelissen/reproducing-actions/.github/workflows/reusable-test.yml@add-logging
```

This change **must** be reverted prior to merging:

```suggestion
    uses: ericcornelissen/reproducing-actions/.github/workflows/reusable-test.yml@main
```